modules/lite/webpack.config.js

```diff
@@ -12,9 +12,8 @@ version = version.match(/^(\d+)\.(\d+)/)[0];
 const config = {
   resolve: {
     alias: {
-      'deck.gl': resolve(ROOT, 'src'),
       'mapbox-gl': resolve(PACKAGE_ROOT, 'utils/mapbox-gl'),
-      './mapbox': resolve(PACKAGE_ROOT, 'node_modules/react-map-gl/src/mapbox/mapbox.js')
+      './mapbox': resolve(ROOT, 'node_modules/react-map-gl/src/mapbox/mapbox.js')
```

Comment:
This is clever. Though I hoped we were also planning to let this module serve as a `@deck.gl/addons` so that standard apps could `import {Mapbox} from `@deck.gl/...`